styles.css

```diff
@@ -66,6 +114,70 @@ Text {
   border-radius: 50px;
   margin-left: 15px;
   margin-right: 10px;
+  height: 50px;
+}
+
+.NewPost {
+  display: flex;
+  flex-direction: column;
+  color: #8899a6;
+}
+
+.PostButtons {
+  display: flex;
+  flex-direction: row;
+  margin-left: 55px;
+  margin-top: 15px;
+  height: 60px;
+  border-top: solid;
+  border-color: rgba(219, 219, 219, 0.13);
+}
+
+.TweetButton2 {
```

Comment:
This is a code smell that a better name should be used.

Also you can move the common styles into a common/base class and use a separate class for any "modifiers":

```css
.button {
  // shared styles
}

.tweetButton {
  margin: 20px; // some modifier
}

.anotherButton {
  margin: 10px;
}
```

An alternative, which I highly recommend as it is the normal practice, is to use [CSS modules](https://github.com/css-modules/css-modules) instead of one big CSS file. This scopes the CSS classes to a component. The effect being that you only need unique classes per a component's CSS module file (Sidebar.module.css).

While you're at it, I would look into using SASS/SASS Modules in place of normal CSS: https://sass-lang.com/